app/models/early_career_payments/dqt_record.rb

```diff
@@ -0,0 +1,114 @@
+module EarlyCareerPayments
+  class DqtRecord
```

Comment:
@nomasprime As requested down in the `app/models/maths_and_physics/dqt_record.rb` file, please include the documentation lines `2-16` and `20-27` (these I would argue as being essential to include) and `73-79` in the relevant places.

When we joined the project months ago we had zero ideas about JACS and HECOS codes. These comments are relevant to the file and should not be hidden in a single file. These are what I would consider +ve comments